rune/dist/Makefile

```diff
@@ -3,6 +3,7 @@ RPMBUILD_DIR := $(shell mktemp -u /tmp/rpmbuild.XXXX)
 RELEASE_TARBALL_URL := https://github.com/alibaba/inclavare-containers/archive/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
 RELEASE_TARBALL_FILE := $(RPMBUILD_DIR)/SOURCES/v$(INCLAVARE_CONTAINERS_VERSION).tar.gz
 RELEASE_TARBALL_EXIST := $(shell if [ -f $(RELEASE_TARBALL_FILE) ]; then echo "y"; else echo "n"; fi;)
+DIST_VERSION := $(shell cat /etc/*-release | grep ^ID=[A-Za-z]* | sed s/ID=//g | sed s/\"//g)
```

Comment:
Call this variable as DISTRO.

Reply:
I think actually we want to know which package manager is employed. If dpkg program exists, deb package manager is used; If rpm program exists, rpm package manager is used.